fideslib/oauth/api/deps.py

```diff
@@ -39,7 +39,7 @@ def oauth2_scheme() -> OAuth2ClientCredentialsBearer:
 
     This should be overridden by the installing package.
     """
-    return OAuth2ClientCredentialsBearer(tokenUrl=TOKEN)
+    return OAuth2ClientCredentialsBearer(tokenUrl=f"/api/v1{TOKEN}")
```

Comment:
The V1 prefix could use the constant we have stored in `urn_registry.py`

https://github.com/ethyca/fidesops/blob/main/src/fidesops/ops/api/v1/urn_registry.py#L2